data/schema/dynamic.proto

```
@@ -0,0 +1,18 @@
+syntax = "proto2";
+package dynamic;
```

Buf (1.66.1)

[error] Files with package "dynamic" must be within a directory "dynamic" relative to root but were in directory "data/schema".

(PACKAGE_DIRECTORY_MATCH)